src/builder/pod/mod.rs

```diff
@@ -25,8 +27,18 @@ use k8s_openapi::{
 };
 use tracing::warn;
 
+#[derive(Debug, thiserror::Error, PartialEq)]
+pub enum Error {
+    #[error("The duration {duration} is too long to put it into the Pods terminationGracePeriodSeconds. The maximum duration is {}", Duration::from_secs(i64::MAX as u64))]
+    DurationTooLong {
```

Comment:
```suggestion
    TerminationGracePeriodTooLong {
```

The error message notes the field specifically, so we should do the same in code to avoid confusion.